### PR TITLE
Update nyaatorrents.py

### PR DESCRIPTION
**Pull request by Nibeck1309**
remove "sort": 2,     # Sort Descending By Seeders because it cause anime ep not found for some language (example VOSTFR)

thanks

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/nyaatorrents.py [hunk 1]
```diff
@@ -63,7 +63,6 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                 params = {
                     "page": 'rss',
                     "cats": '1_0',  # All anime
-                    "sort": 2,     # Sort Descending By Seeders
```
Comment (miigotu) [inline sickbeard/providers/nyaatorrents.py:66]: Can only sort by seeders when mode != 'RSS', RSS needs to sort by date.

**Comment by duramato** [comment]
@Nibeck1309 Are you going to make the changes as propused?

**Comment by fernandog** [comment]
@Nibeck1309 
2 = seeders. so how to sort by date?

**Comment by Nibeck1309** [comment]
you can use 

"cats": '1_0',  # All anime
"sort": '1',      # By Date
"order":'1'     # Descending

**Comment by fernandog** [comment]
@Nibeck1309 if "sort" is not informed, what is default sort setting?

**Comment by Nibeck1309** [comment]
@fernandog if sort is not informed, the default sort setting is by Date

But if you absolutly use the sort setting, you can use this line :

"cats": '1_0', # All anime
"sort": '1', # By Date
"order":'1' # Descending

**Comment by Nibeck1309** [comment]
@fernandog and @miigotu you can use other setting for limiting number of file result  

"minage":'0',        #minimal age of file
"maxage":"90",   #max age of file

**Comment by fernandog** [comment]
The code should be update to use date sort when RSS and seed sort when not RSS

@miigotu agree?

**Comment by miigotu** [comment]
And the category based on ep_obj.is_anime or w/e when not on rss mode.

**Comment by fernandog** [comment]
nyaatorrents is anime only isnt?

**Comment by Nibeck1309** [comment]
yes, nyaatorrent is for anime only

**Comment by fernandog** [comment]
So what are those categories?!

**Pull request closed without merging**